### DISCUSSION TITLE
Reason for default reset's ?

### DISCUSSION THREAD

**Discussion by vgr28** [tbnobody/OpenDTU#536] [Q&A]
Hallo, was sind denn die Ursachen für solche resets (alle Werte) ? Die Entfernung zwischen DTU und WR ist nur 30-40cm? Stromversorgung kann es auch nicht sein (5A). Kondensator habe ich noch keinen verbaut.

![2023-01-21_14h27_42](https://user-images.githubusercontent.com/28902513/214268983-5cd7342c-de45-4b9a-8b1c-ddbdb6f43665.png)

**Comment by CONSULitAS** [comment]
> Kondensator habe ich noch keinen verbaut.

Dann hast Du ja sicher das gelesen: https://github.com/tbnobody/OpenDTU#troubleshooting

Und dann hilft ja auch sicher der Kondensator.
